packages/terra-base/src/Base.jsx

```diff
@@ -87,14 +87,14 @@ class Base extends React.Component {
 
     if (locale === undefined) {
       return (
-        <div {...customProps}>
+        <div className={['base', customProps.className]} {...customProps}>
           {renderChildren}
         </div>);
     }
 
     if (!this.state.areTranslationsLoaded) return <div>{this.props.translationsLoadingPlaceholder}</div>;
     return (
-      <I18nProvider {...customProps} locale={this.state.locale} messages={messages}>
+      <I18nProvider {...customProps} className={['base', customProps.className]} locale={this.state.locale} messages={messages}>
```

Comment:
Let's update this to:
```suggestion
      <I18nProvider data-terra-base {...customProps} locale={this.state.locale} messages={messages}>
```